src/components/CardJob.jsx

```diff
@@ -41,10 +44,20 @@ function CardJob(props) {
       </div>
       <hr />
       <div className={style.btn_block}>
-        {<ButtonLanguage key={role}>{role}</ButtonLanguage>}
-        {<ButtonLanguage key={level}>{level}</ButtonLanguage>}
+        {
+          <ButtonLanguage key={role} handleSelect={handleSelect}>
+            {role}
+          </ButtonLanguage>
+        }
+        {
+          <ButtonLanguage key={level} handleSelect={handleSelect}>
+            {level}
+          </ButtonLanguage>
+        }
```

Comment:
Why do we need curly brackets around these components?

Why do we need key prop for ButtonLanguage?